R/db2.R

```diff
@@ -6,7 +6,7 @@
 ## namespace/module feature so that implementation details can be
 ## hidden away a bit further.
 
-ORDERLY_SCHEMA_VERSION <- "0.0.4"
+ORDERLY_SCHEMA_VERSION <- "0.0.5"
```

Comment:
Can you increase this to 0.0.6 and the package version to 0.5.17 to leap over the current versions? (don't forget to update the news version too)